src/uploads/imageResizer.ts

```diff
@@ -77,6 +77,10 @@ export default async function resizeAndSave({
         resized = resized.toFormat(desiredSize.formatOptions.format, desiredSize.formatOptions.options);
       }
 
+      if ('trimOptions' in desiredSize) {
```

Comment:
The condition checks whether the `trimOptions` key exists.

This seems like the simplest and cleanest approach. Sharp trim options can accept a number, string or object. For example, `0` is a valid configuration that has the following effect:

> Trim pixels with the exact same colour as that of the top-left pixel.

https://sharp.pixelplumbing.com/api-resize#trim